docs/creating-claudie-backup/creating-claudie-backup.md

```diff
@@ -16,6 +16,111 @@ These are the only services that will have a PVC attached to it, the other are s
 
 ## Backing up Claudie
 
+### Manual backup
+
+Claudie is already deployed on an existing Management Cluster and at least 1 Input Manifest has been applied.
+
+Create a directory where the backup of the state will be stored.
+
+```bash
+mkdir claudie-backup
+```
+
+Put your Claudie inputmanifests into the created folder.
```

Comment:
It's not fully clear to me what should I do here. Should I do `kubectl get InputManifest -A -oyaml > ./claudie-backup`?

Reply:
Yes. or if you have them on the file system